Externals/glslang/glslang/MachineIndependent/Constant.cpp

```diff
@@ -832,7 +832,7 @@ TIntermTyped* TIntermediate::fold(TIntermAggregate* aggrNode)
             newConstArray[2] = childConstUnions[0][0] * childConstUnions[1][1] - childConstUnions[0][1] * childConstUnions[1][0];
             break;
         case EOpFaceForward:
-            // If dot(Nref, I) < 0 return N, otherwise return �N:  Arguments are (N, I, Nref).
+            // If dot(Nref, I) < 0 return N, otherwise return �N:  Arguments are (N, I, Nref).
```

Comment:
no idea what symbol this was ever supposed to be, but vscode decided to make it an official garbage symbol